### ISSUE TITLE
Broken pointAtObjectServer

### ISSUE THREAD

**Issue by jgvictores** [roboticslab-uc3m/teo-head#65]
`pointAtObjectServer`, recently imported from asibot-main, is broken. See current `import-pointAtObjectServer`branch.

**Comment by jgvictores** [comment]
Error is:
```
error: ‘class vtkPolyDataMapper’ has no member named ‘SetInput’
```
Help at: http://www.vtk.org/Wiki/VTK/VTK_6_Migration/Replacement_of_SetInput

...which reminds us of: https://github.com/roboticslab-uc3m/QA/issues/7 and https://github.com/roboticslab-uc3m/QA/issues/14

**Comment by jgvictores** [comment]
Fixed at 41723ae87990a8fe8c6d958d8b465d881932120d, opened #66 for documentation.
